wynnset/tapestry-wp
# Add custom audio recorder

Closes #422

- [x] Hook up recorder to backend
- [x] Update authoring tool to a checkbox instead of combobox for audio type answers
- [x] Make sure that audio based entry objects still work as expected

**Testing**

- Run `npm install` to install the new recorder polyfill required for older browsers
- Create a new quiz with an audio recorder as one of the answer types
- Open the quiz then record something
- The same recording should appear when the page is refreshed and the quiz reopens
